js_basics/fizz_buzz/index.js

```diff
@@ -0,0 +1,16 @@
+const args = process.argv.slice(2)
+const first = "fizz"
+const second = "buzz"
+if(args){
+    for(let i= 1;i<101;i++){
+        if(((i)%parseInt(args[0])===0) && ((i)%parseInt(args[1])===0)){
+            console.log(first+second)
```

Comment:
I would directly put the strings "fizz" and "buzz" into the console.logs without the need of a variable.